### PR TITLE
Fixes #13626 - Enable ostree on capsules

### PR DESCRIPTION
**Pull request by parthaa**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -46,6 +46,9 @@
 #
 # $qpid_router_broker_port::            Port of qpidd broker to connect to
 #
+# $enable_ostree::                      Boolean to enable ostree plugin. This requires existence of a ostree install.Defaults to false
```
Comment (ehelms) [inline manifests/init.pp]: Mention of defaults not needed as that is handled by the value in the params file

File: manifests/init.pp [hunk 2]
```diff
@@ -70,7 +73,9 @@
   $qpid_router_agent_port    = $capsule::params::qpid_router_agent_port,
   $qpid_router_broker_addr   = $capsule::params::qpid_router_broker_addr,
   $qpid_router_broker_port   = $capsule::params::qpid_router_broker_port,
+  $enable_ostree              = $capsule::params::enable_ostree,
```
Comment (ehelms) [inline manifests/init.pp]: Alignment here please -- makes it look so pretty

File: manifests/init.pp [hunk 3]
```diff
@@ -46,6 +46,9 @@
 #
 # $qpid_router_broker_port::            Port of qpidd broker to connect to
 #
+# $enable_ostree::                      Boolean to enable ostree plugin. This requires existence of a ostree install.
```
Comment (ekohl) [inline manifests/init.pp]: Nitpick: a**n** ostree install.

**Comment by ehelms** [comment]
ACK - thanks @parthaa !

**Pull request merged**